XpressAI/arrow
# the latest upstream + the latest NEC works

the latest upstream + the latest NEC changes
Arrow NEC:
- fix benchmark builds
- fix IntToString failure: the inline function was aggressively optimized. it was rewritten with template
- change: nearbyint to rint which works correctly
- fix some arrow/parquet compile/test errors
- fix/replace unrecognized U string literal with int
- disable decimal tests as it causes SIGSEGV . (probably, many template instantiations in one translation unit)
- fix linking paths. there was a problem in linking as library paths were added hardcoded. we fixed it using mini-hack as policies and other settings did not work as intended
- fix debug build failures. it should build boost properly
- fix cmake thirdparty libs section on NEC
- arrow test failures reduced with -mno-vector. (we could re-enable it selectively later for better performance)
- fix/workaround nc++ bug: struct type with bool type and bit fields incorrect behavior
- add aurora.cmake and fix cmake related errors
- add Nec.md
